lib/livebook_web/channels/socket.ex

```diff
@@ -1,26 +1,19 @@
 defmodule LivebookWeb.Socket do
   use Phoenix.Socket
 
-  # App channels
   channel "js_output", LivebookWeb.JSOutputChannel
 
-  # LiveView channels
-  channel "lvu:*", Phoenix.LiveView.UploadChannel
-  channel "lv:*", Phoenix.LiveView.Channel
-
   @impl true
-  def connect(params, socket, info) do
+  def connect(_params, socket, info) do
     auth_mode = Livebook.Config.auth_mode()
 
     if LivebookWeb.AuthPlug.authenticated?(info.session || %{}, info.uri.port, auth_mode) do
-      Phoenix.LiveView.Socket.connect(params, socket, info)
+      {:ok, socket}
     else
       :error
     end
   end
 
   @impl true
-  def id(socket) do
-    Phoenix.LiveView.Socket.id(socket)
-  end
+  def id(_socket), do: nil
```

Comment:
I would keep the same ID because we want to disconnect it too?

Reply:
Ah, in order to use `Phoenix.LiveView.Socket.id(socket)` we need to call `Phoenix.LiveView.Socket.connect(socket)` above too.

Reply:
We are not setting `:live_socket_id`, so it's `nil` in both cases. We never disconnect, because I don't think we currently have a scenario where we would want to disconnect?

Reply:
Got it!